be/src/service/internal_service.cpp

```diff
@@ -558,15 +558,19 @@ void PInternalServiceImpl::cancel_plan_fragment(google::protobuf::RpcController*
         tid.__set_lo(request->finst_id().lo());
         signal::set_signal_task_id(tid);
         Status st = Status::OK();
-        if (request->has_cancel_reason()) {
-            LOG(INFO) << "cancel fragment, fragment_instance_id=" << print_id(tid)
-                      << ", reason: " << PPlanFragmentCancelReason_Name(request->cancel_reason());
-            _exec_env->fragment_mgr()->cancel_instance(tid, request->cancel_reason());
-        } else {
-            LOG(INFO) << "cancel fragment, fragment_instance_id=" << print_id(tid);
-            _exec_env->fragment_mgr()->cancel_instance(tid,
-                                                       PPlanFragmentCancelReason::INTERNAL_ERROR);
-        }
+
+        const bool is_pipeline_fragment = _exec_env->fragment_mgr()->is_pipeline_fragment(tid);
+        const bool has_cancel_reason = request->has_cancel_reason();
+        LOG(INFO) << fmt::format("Cancel {}instance {}, reason: {}",
+                                 is_pipeline_fragment ? "pipeline " : "", print_id(tid),
+                                 has_cancel_reason
+                                         ? PPlanFragmentCancelReason_Name(request->cancel_reason())
+                                         : "INTERNAL_ERROR");
+
+        _exec_env->fragment_mgr()->cancel_instance(
```

Comment:
rename cancel_instance to  cancel_fragment_instance

Reply:
done